bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -252,19 +250,20 @@ private func dummy() {
     _ = NSLocalizedString("Select a color", comment: "")
     _ = NSLocalizedString("Select a color above", comment: "")
     _ = NSLocalizedString("Select a layout", comment: "")
-    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select control option value e.g: \"Auto, 25%\".")
+    _ = NSLocalizedString("Selected: %s", comment: "translators: %s: Select font size option value e.g: \"Selected: Large\".\ntranslators: %s: Select control option value e.g: \"Auto, 25%\".")
+    _ = NSLocalizedString("Selected: Default", comment: "")
     _ = NSLocalizedString("Set as Featured Image ", comment: "")
     _ = NSLocalizedString("Show post content", comment: "")
     _ = NSLocalizedString("Slash inserter results", comment: "translators: Slash inserter autocomplete results")
     _ = NSLocalizedString("Some blocks have additional settings. Tap the settings icon on the bottom right of the block to view more options.", comment: "")
-    _ = NSLocalizedString("Sorting and filtering", comment: "")
```

Comment:
- Removal
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/latest-posts/edit.js#L330:L330)

This string is already translated as it's used in the web version, so we don't need to include it here.